mycroft/util/network_utils.py

```diff
@@ -46,18 +55,27 @@ def _connected_dns(host="8.8.8.8", port=53, timeout=3):
     # Host: 8.8.8.8 (google-public-dns-a.google.com)
     # OpenPort: 53/tcp
     # Service: domain (DNS/TCP)
+    config = get_network_tests_config()
+    if host is None:
+        host = config.get('dns_primary')
     try:
         s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         s.settimeout(timeout)
         s.connect((host, port))
         return True
     except IOError:
+        LOG.error("Unable to connect to primary DNS server, "
+                  "trying secondary...")
         try:
             s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
             s.settimeout(timeout)
-            s.connect(("8.8.4.4", port))
+            dns_secondary = config.get('dns_secondary')
+            LOG.info(config)
+            LOG.info(dns_secondary)
```

Comment:
These looks like debug outputs? I'd recommend removing them or providing some describing text around the values.

Reply:
oops, they are stray debuging will remove